src/catalog/src/lib.rs

```diff
@@ -106,6 +107,7 @@ pub trait CatalogManager: Send + Sync {
         catalog: &str,
         schema: &str,
         table_name: &str,
+        table_id: &uint64,
```

Comment:
I think `table_id` should not put here.

Reply:
@MichaelScofield should 
I remove it from there?

Reply:
yes